### DISCUSSION TITLE
关于增加快捷键的建议

### DISCUSSION THREAD

**Discussion by Dragon-Zhuang** [obsidianzh/forum#69] [Ideas想法]
本人刚刚从Typora转过来，被obsidian强大的internal link所折服！
这里提几个关于快捷键的建议，都是来自于Typora的启发。
1. 代码块的插入：这个功能功能对于刚需者来说十分关键，否则需要每次按同一个键6次来实现这个功能，是十分不方便的；
2. 公式块的插入：既包括行内公式，也包括行间公式，在中文打字时，需要先切换为英文输入，再输入$符号，也是十分不方便的；
3. 标题的添加：Typora中采用ctrl加数字的方式来实现标题的添加，十分方便高效，同时希望能够加入对标题编号的功能；
以上的问题如果大家知道如何设置以实现，希望大家不吝赐教！
祝obsidian越来越好！

**Comment by renmu123** [comment]
1. you can search "Code block from selection" in plugin store，or clone this [plugin](https://github.com/renmu123/obsidian-code-block-from-selection)（fork from "Code block from selection", and l just add some features）
3. you can serach "Format Hotkeys" in plugin store

我才发现是中文社区
1. 可以在商店搜索 "Code block from selection"  这个插件，或者可以clone我刚修改的项目，三十分钟前刚加了一些新的功能
3. 在商店中搜索 “Format Hotkeys” 插件，但是也没有 ctrl+ 快捷键
